invenio_records_rest/__init__.py

```diff
@@ -22,7 +22,634 @@
+...    # ...,
+... )
+
+Now that we have configured the endpoint as we want, we can go ahead and install
+the extension with:
+
+>>> ext_records_rest = InvenioRecordsREST(app)
+
+Finally we can see that the endpoints have been registered:
+
+>>> app.url_map
+Map([<Rule '/records/_options' (OPTIONS, HEAD, GET) -> invenio_records_rest.recid_list_options>,
+ <Rule '/records/' (POST, OPTIONS, HEAD, GET) -> invenio_records_rest.recid_list>,
+ <Rule '/records/<pid_value>' (OPTIONS, DELETE, HEAD, PATCH, PUT, GET) -> invenio_records_rest.recid_item>,
+ <Rule '/static/<filename>' (OPTIONS, HEAD, GET) -> static>])
+
+Demo data
+~~~~~~~~~
+Once you have this code you can run the application and create some records:
+
+.. code-block:: console
+
+ $ curl -L -H 'Content-Type:application/json' -d '{"title": "Test Record 1"}' -XPOST localhost:5000/records
+ $ curl -L -H 'Content-Type:application/json' -d '{"title": "Test Record 2"}' -XPOST localhost:5000/records
+
+Searching
+---------
+Invenio-Records-REST offers a rich set of fundamental search operations using
+query strings, filters, sorting and pagination and also more advanced
+operations including aggregations and suggestions.
+
+``/records/?q=&size=10&page=1&sort=bestmatch&type=test``
+
+Faceted Search
+~~~~~~~~~~~~~~
+
+Faceted search is enabled through filters and aggregations. It can be used to
+obtain subsets of documents which meet certain criteria, group results by
+categories, and provide an easy to navigate faceted menu.
+
+``/records/?type=test&type=anoterhvalye/``
+
+
+Aggregations
+++++++++++++
+By exposing the Elasticsearch API by default, we can use the advanced
+features it provides, such as the `aggregations framework
+<https://www.elastic.co/guide/en/elasticsearch/reference/5.6/search-aggregations.html>`_.
+
+These features include:
+
+- `bucketing <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/search-aggregations-bucket.html>`_
+  "used to group the documents by a certain criterion".
+- `metric <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/search-aggregations-metrics.html>`_
+  "used to create different types of metrics from values extracted from the documents
+  being aggregated".
+- `matrix <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/search-aggregations-matrix.html>`_
+  "used to produce a matrix result based on the values extracted from the requested document fields".
+- `pipeline <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/search-aggregations-pipeline.html>`_
+  "used to aggregate the output of other aggregations and their associated metrics".
+
+
+Filters
++++++++
+Filters can be applied at different points of a query having different effects
+for aggregations, but not for search queries.
+
+
+To set the aggregations and filters you want you can modify the
+`RECORDS_REST_FACETS`.
+
+>>> from .facets import terms_filter
+>>> app.config['RECORDS_REST_FACETS'] = {
+...     'index_name': {
```

Comment:
Maybe make make a comment here emphasizing that the name of the index/alias (ie. `record`) and not of the namespace (ie. `recid`) has to be used

Reply:
Good point, thanks